asab/library/providers/filesystem.py

```diff
@@ -31,6 +32,13 @@ def __init__(self, library, path, layer, *, set_ready=True):
 		'''
 
 		super().__init__(library, layer)
+		# Parse file:// URI if present
+		if path.startswith('file:///'):
+			parsed_uri = urllib.parse.urlparse(path)
+			# Extract the absolute path directly
+			path = parsed_uri.path
+		elif path.startswith('file://'):
+			raise ValueError("Invalid file URL: Only 'file:///' absolute paths are supported")
```

Comment:
I've been discussing this lately with Otto. Almost every service is reading from the library. If you misconfigure library in the UI, hit Apply, the whole stack goes down because of that and you lose possibility to fix it. 

I'd like to see the service starting with error saying that the library was misconfigured and the content is not accessible rather than the whole container going down. (It means all asab services restarting wildly.)

Reply:
There must be a validation during UI save process ... anything later is just too late.

Reply:
It should be just a warning, not exit of the app.

Reply:
I agree - this MUST be a warning only, not the hard exit of the application @mithunbharadwaj